recipes/LibriSpeech/ASR/transformer/hparams/conformer_large_rope.yaml

```diff
@@ -0,0 +1,338 @@
+# ############################################################################
```

Comment:
This guy looks VERY similar to our conformer_large.yaml, right? Why not changing the main one with RoPE. The model is just better, so I don't see any problem with that.